### PR TITLE
Feed header UI enhancement and adv search

### LINKED ISSUE #438: Replace the media wall toggle from feed with a simple button
* Currently there is an unnecessary drop down to activate the media walls
* Remove this two step process.
* Make a single toggle button

### PR DESCRIPTION
**Pull request by hemantjadon**
* Improve UI of feed header component

* Make The feed header A11y compatible

* Add Advanced search features to the results

* Add location based search

* Add time restricted search

https://hemantjadon.github.io/loklak_search

**Closes #438, #43**

### CODE REVIEW

**Comment by mariobehling** [comment]
Please add a screenshot and point out what is different now.

**Comment by hemantjadon** [comment]
@mariobehling I have updated the link.

File: src/app/feed/feed-advanced-search/feed-advanced-search.component.html [hunk 1]
```diff
@@ -1,11 +1,86 @@
...
+						<button md-menu-item value="UK"
+							(click)="getLocationBasedResults($event.currentTarget.value)">
+							Country: UK
+						</button>
+					</md-menu>
+				</div>
+			</div>
+			<div class="message"
+				*ngIf="!(isSearching$ | async) && (areResultsAvailable$ | async)">
+				Loklak scrapes social media messages
```
Comment (Achint08) [inline src/app/feed/feed-advanced-search/feed-advanced-search.component.html]: Can this be replaced by the `result count`?

File: src/app/feed/feed-advanced-search/feed-advanced-search.component.ts [hunk 2]
```diff
@@ -18,44 +25,144 @@ import { countrycodearray } from '../../shared/countrycode/countrycode';
...
 	}
 
-	public getVideoResults() {
-		this.filterList = {
-			image: false,
-			video: true
-		};
-		this.selectedTab = 'video';
+	public getFilterResults(value: string) {
+		console.log(value);
```
Comment (Achint08) [inline src/app/feed/feed-advanced-search/feed-advanced-search.component.ts]: Remove console log

File: src/app/feed/feed-advanced-search/feed-advanced-search.component.html (hunk 1 above)
Comment (hemantjadon) [inline src/app/feed/feed-advanced-search/feed-advanced-search.component.html]: Yeah acutually I needed suggestions for this. What should be shown up here? Result count can be shown then again it becomes tricky as server count varies so I was confused. I thought to enter here what loklak server is doing.

**Comment by mariobehling** [comment]
Good job!

**Pull request merged**